android-components/components/feature/fxsuggest/src/main/java/mozilla/components/facts/FxSuggestFacts.kt

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.fxsuggest.facts
+
+import mozilla.components.feature.fxsuggest.FxSuggestClickInfo
+import mozilla.components.support.base.Component
+import mozilla.components.support.base.facts.Action
+import mozilla.components.support.base.facts.Fact
+import mozilla.components.support.base.facts.collect
+
+/**
+ * Facts emitted for telemetry related to the Firefox Suggest feature.
+ */
+class FxSuggestFacts {
+    /**
+     * Specific types of telemetry items.
+     */
+    object Items {
+        const val SPONSORED_SUGGESTION_CLICKED = "sponsored_suggestion_clicked"
```

Comment:
We learned from Desktop that we'd have many kinds of sponsored suggestions including the AMP ones and they don't share the same metric schema, so maybe we can make it more specific:

```
    object Events {
        const val AMP_SUGGESTION_CLICKED = "amp_suggestion_clicked"
        const val AMP_SUGGESTION_VIEWED = "amp_suggestion_viewed"
        // For another partner
        const val YELP_SUGGESTION_CLICKED = "yelp_suggestion_clicked"
        ...
```

Reply:
renamed to AMP_SUGGESTION_CLICKED, and I guess we will add them as we add the other things?